### PR TITLE
MGDOBR-63: Ansible Gateway for self-healing infrastructure demo

### PR DESCRIPTION
**Pull request by kostola**
[Jira task: MGDOBR-63](https://issues.redhat.com/browse/MGDOBR-63) 

Please make sure that your PR meets the following requirements:

- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [x] All new functionality is tested and uses org.assertj.core.api.Assertions for Assertions
- [x] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: use-cases/pom.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <groupId>com.redhat.service.bridge</groupId>
+  <artifactId>use-cases</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Event Bridge :: Use Cases</name>
+  <description>Use cases for event bridge</description>
+
+  <modules>
+    <module>self-healing-infrastructure</module>
```
Comment (danielezonca) [inline use-cases/pom.xml:15]: Let's rename it as ansible-tower-integration for now.

File: use-cases/self-healing-infrastructure/ansible-gateway/pom.xml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-arc</artifactId>
+    </dependency>
+<!--    <dependency>-->
+<!--      <groupId>io.quarkus</groupId>-->
+<!--      <artifactId>quarkus-resteasy</artifactId>-->
+<!--    </dependency>-->
```
Comment (danielezonca) [inline use-cases/self-healing-infrastructure/ansible-gateway/pom.xml:56]: To be removed

File: use-cases/self-healing-infrastructure/pom.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.redhat.service.bridge</groupId>
+  <artifactId>self-healing-infrastructure</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Event Bridge :: Self-Healing Infrastructure Use Case</name>
+  <description>Self-healing infrastructure use case for event bridge</description>
```
Comment (danielezonca) [inline use-cases/self-healing-infrastructure/pom.xml:12]: See previous comment

File: use-cases/self-healing-infrastructure/pom.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (danielezonca) [inline use-cases/self-healing-infrastructure/pom.xml:1]: Please add a readme about how to install/run it (+ docker-compose if necessary)

File: use-cases/pom.xml (hunk 1 above)
Comment (kostola) [reply use-cases/pom.xml]: Done :+1:

**Pull request merged**